src/sentry/static/sentry/app/components/timeSince.tsx

```diff
@@ -142,18 +142,20 @@ function getDateObj(date: RelaxedDateType): Date {
   return date;
 }
 
-function getRelativeDate(
+export function getRelativeDate(
   currentDateTime: RelaxedDateType,
   suffix?: string,
   shorten?: boolean
 ): string {
   const date = getDateObj(currentDateTime);
 
-  if (shorten) {
+  if (shorten && suffix) {
     return t('%(time)s %(suffix)s', {
       time: getDuration(moment().diff(moment(date), 'seconds'), 0, true),
       suffix,
     });
+  } else if (shorten && !suffix) {
+    return getDuration(moment().diff(moment(date), 'seconds'), 0, true);
```

Comment:
alternatively if you don't want to rewrite `getDuration(moment().diff(moment(date)...` you can change the way the if statements are structured to be like

```js
if (shorten) {
  const duration = getDuration(moment().diff(moment(date), 'seconds'), 0, true);
  
  if (suffix) {
    return t('%(time)s %(suffix)s', {
      time: duration,
      suffix,
    });
  } else {
    return duration;
  }
} else if (!suffix) {
  ...
} else if (suffix === 'ago') {
  ...
} else if (suffix === 'old') {
  ...
} else {
  ...
}
```

Reply:
I didn't want to nest if statements, it becomes very unreadable